### PR TITLE
✨ RedisDestination FE

### PR DESCRIPTION
**Pull request by RTLS**

### CODE REVIEW

**Comment by RTLS** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/559?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#559** <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/559?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/559?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#557** <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/557?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#556** <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/556?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#555** <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/555?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: assets/svelte/consumers/DestinationRedisForm.svelte [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+              <Label for="max-ack-pending">Max ack pending</Label>
+              <Input
+                id="max-ack-pending"
+                type="number"
+                bind:value={form.maxAckPending}
+              />
+              <p class="text-sm text-muted-foreground">
+                Sets the maximum number of messages that can be pending
+                acknowledgment at any time. This helps control the flow of
+                messages and prevents overwhelming the consumer.
```
Comment (acco) [inline assets/svelte/consumers/DestinationRedisForm.svelte:113]: Let's change this? (I probably missed for SQS):

> Sets the maximum number of messages that can be in-flight to Redis
                at any time. This helps control the flow of
                messages and prevents overwhelming Redis.

File: lib/sequin/redis/client.ex [hunk 2]
```diff
@@ -10,6 +10,8 @@ defmodule Sequin.Redis.Client do
 
   @impl Sequin.Redis
   def send_messages(%RedisDestination{} = destination, messages) do
+    dbg()
```
Comment (acco) [inline lib/sequin/redis/client.ex:13]: dbg

File: lib/sequin_web/live/destination_consumers/index.ex [hunk 3]
```diff
@@ -127,6 +128,18 @@ defmodule SequinWeb.DestinationConsumersLive.Index do
...
   end
 
+  defp render_consumer_form(%{form_kind: "redis"} = assigns) do
+    ~H"""
+    <.live_component
+      current_user={@current_user}
+      module={ConsumerForm}
+      id="new-consumer"
+      action={:new}
+      consumer={%DestinationConsumer{type: :redis, batch_size: 100, destination: %RedisDestination{}}}
```
Comment (acco) [inline lib/sequin_web/live/destination_consumers/index.ex:138]: Awesome, and good call on setting a high default for batch_size 👍

**Review by acco — APPROVED** [review]

**Pull request merged**